invenio_files_rest/cli.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Click command-line interface for file management."""
+
+from __future__ import absolute_import, print_function
+
+import os
+
+import click
+from flask import current_app
+from flask_cli import with_appcontext
+from invenio_db import db
+
+#
+# File management commands
+#
+
+
+@click.group()
+def files():
+    """File management commands."""
+
+
+@files.command()
+@click.argument('source', type=click.Path(exists=True, resolve_path=True))
+@click.option('--checksum/--no-checksum', default=False)
+@with_appcontext
+def bucket_from(source, checksum):
```

Comment:
Would it perhaps make sense to split it in two commands into:

- ``bucket`` (create a bucket)
- ``cp <source> <bucket, key?>`` (then we could later extend with ``cp <bucket, key?> <destination>`` as well as ``cp <src bucket, key> <dest bucket, key>``) so we have basic upload/download cli.

I.e. basically imitating unix commands.